packages/js-config-webpack-plugin/package.json

```diff
@@ -51,17 +51,16 @@
     "update-snapshots": "jest -u --runInBand"
   },
   "peerDependencies": {
-    "@babel/core": ">=7.0.0",
     "@babel/preset-env": ">=7.0.0",
     "@babel/preset-react": ">=7.0.0",
     "webpack": ">=4.36.0"
   },
   "dependencies": {
+    "@babel/core": "7.9.0",
     "babel-loader": "^8.0.0-beta.4",
     "thread-loader": "^2.1.1"
   },
   "devDependencies": {
-    "@babel/core": "7.2.2",
     "@babel/preset-env": "7.2.3",
```

Comment:
If we use your proposed way of https://github.com/tj/co/blob/717b043371ba057cb7a4a2a4e47120d598116ed7/index.js#L221 we don't need to release a change which forces people to upgrade their babel setup.

So we can revert this change and might only upgrade babel in the dev dependencies to 7.9

Reply:
Done :+1: